src/solvers/dgsem_tree/dg_2d_subcell_limiters.jl

```diff
@@ -466,22 +472,40 @@ end
 end
 
 """
-    get_boundary_outer_state(boundary_condition::BoundaryConditionDirichlet,
-                             cache, t, equations, dg, indices...)
+    get_boundary_outer_state(u_inner, t,
+                             boundary_condition::BoundaryConditionDirichlet,
+                             orientation_or_normal, direction,
+                             mesh, equations, dg, cache, indices...)
```

Comment:
The elixir supersonic cylinder uses other boundary conditions than periodic or Dirichlet. So, a few more information are needed here as the inner value `u_inner`, `orientation_or_normal` and `direction` (for TreeMesh and StructuredMesh).